internal/api/v1/stage.go

```diff
@@ -175,20 +174,20 @@ func createCertificates(ctx context.Context, cfg *rest.Config, app gitea.App) er
 	// Create production certificate if it is provided by user
 	// else create a local cluster self-signed tls secret.
 	if !strings.Contains(c.Domain, "omg.howdoi.website") {
-		err = createProductionCertificate(ctx, cfg, app, c.Domain)
+		err = createCertificate(ctx, cfg, app, c.Domain, "letsencrypt-production")
 		if err != nil {
 			return errors.Wrap(err, "create production ssl certificate failed")
 		}
 	} else {
-		err = createLocalCertificate(ctx, cfg, app, c.Domain)
+		err = createCertificate(ctx, cfg, app, c.Domain, "selfsigned-issuers")
```

Comment:
Shouldn't this match this line: https://github.com/epinio/epinio/pull/385/files#diff-765c4fca879b550830d6c022bb21b9d959b31ca945b7b17853f64fc6fe1006deR244 ?

`selfsigned-issuers` -> `selfsigned-issuer` ?

Reply:
fixed